sapi/fpm/fpm/fpm_events.h

```diff
@@ -43,7 +43,7 @@ int fpm_event_init_main(void);
 int fpm_event_set(struct fpm_event_s *ev, int fd, int flags, void (*callback)(struct fpm_event_s *, short, void *), void *arg);
 int fpm_event_add(struct fpm_event_s *ev, unsigned long int timeout);
 int fpm_event_del(struct fpm_event_s *ev);
-int fpm_event_pre_init(char *machanism);
+int fpm_event_pre_init(char *mechanism);
 const char *fpm_event_mechanism_name(void);
```

Comment:
Careful review please.

Reply:
I think it's OK, it was already fixed in 462da6e09c2eb63aeb9ba357c659d490a9bc46d9 but then probably an erroneus conflict resolution during merge of e2a5428ca08ad10c0f08a686e974af774b4d1b18 in e2a5428ca08ad10c0f08a686e974af774b4d1b18 turned it back to 'machanism', that's why the corresponding variable in the .c file is already correct.